requirements-cli.txt

```diff
@@ -5,3 +5,4 @@ py-cpuinfo
 pytest>=7.0.0
 PyYAML
 requests
+sigfig
```

Comment:
Adding a new dependency to the CLI will require some changes to https://github.com/conda-forge/conbench-feedstock too, but I'm not sure of the exact process. @ElenaHenderson might know. This is an underdocumented aspect of the CLI right now.

Either way, I think bumping the version in setup.py is a good idea.

Reply:
Interesting. https://github.com/conda-forge/conbench-feedstock/pull/8 is a template maybe.